configs/components/fesom/fesom-2.1.yaml

```
@@ -201,52 +200,47 @@ restart_out_sources:
         fesom_bin_restart_info: fesom_bin_restart/*.info
         fesom_bin_restart: fesom_bin_restart/np${nproc}/*
         wiso_restart: fesom.${end_date!syear}.wiso.restart.nc
-        icb_restart: iceberg.restart #.${parent_date!syear}
-        icb_restart_ISM: iceberg.restart.ISM
+        icb_restart: iceberg.restart.${end_date!syear} #_${start_date!syear!smonth!sday}-${end_date!syear!smonth!sday} #.${parent_date!syear}
+        icb_restart_ISM: iceberg.restart.ISM.${end_date!syear} #_${start_date!syear!smonth!sday}-${end_date!syear!smonth!sday}
```

Remove the comments if it works

```suggestion
        icb_restart: iceberg.restart.${end_date!syear}
        icb_restart_ISM: iceberg.restart.ISM.${end_date!syear}
```